src/Controller/Activity/ActivityController.php

```diff
@@ -45,15 +45,15 @@ public function __construct(EventDispatcherInterface $events, EntityManagerInter
 
     /**
      * Lists all activities.
-     *
-     * @MenuItem(title="Terug naar frontend", menu="admin-profile", class="mobile")
-     * @MenuItem(title="Activiteiten")
-     * @Route("/", name="index", methods={"GET"})
      */
+    #[MenuItem(title: "Terug naar frontend", menu: "admin-profile", class: "mobile")]
+    #[MenuItem(title: "Activiteiten")]
+    #[Route("/", name: "index", methods: ["GET"])]
     public function indexAction(): Response
     {
         $groups = [];
-        if ($user = $this->getUser()) {
+        if (null !== $user = $this->getUser()) {
```

Comment:
Within this PR, I tried to replace `$this->getUser()` with `#[CurrentUser]` (see d92865d32c64e882954b95d789ecbf55ff9b9fc2). During that process, I resolved a couple of typing issues. However, that new `#[CurrentUser]` argument was typehinted with the `LocalAccount` type, which is a Doctrine entity type as well. This triggered the Sensio FrameworkExtraBundle to try to populate that value using a route parameter, which failed since there is no relevant route parameter ofcourse. Therefore, I reverted the change in 541583c6258f71235ee9f0b78ebd05e8e0deb2f3.

There, I had to make a decision on whether to revert the typing fixes. My reasoning was as follows: these type fixes are a bit out of scope, I agree, but I feel their presence in this PR doesn't degrade the readability of this PR. Therefore, I opted to leave them in.

Reply:
Alright in that case, they are fine changes to leave within this PR